core/modules/layout/includes/layout.class.inc

```diff
@@ -788,7 +788,7 @@ class Layout {
    * @return LayoutContext[]
    */
   function getContexts($include_types = LayoutContext::USAGE_TYPE_ALL) {
-    $contexts_cache = &backdrop_static(__FUNCTION__, array());
+    $contexts_cache = &backdrop_static(__METHOD__, array());
```

Comment:
When/if this gets merged, we will need to find instances of `backdrop_static_reset()` that use the `__FUNCTION__`  name (`getContexts`) instead of the `__METHOD__` name (`Layout::getContexts`) and change them.

I know for certain that one such instance was just merged in https://github.com/backdrop/backdrop/pull/4085